dash_dangerously_set_inner_html/__init__.py

```diff
@@ -12,15 +13,22 @@
 
 _this_module = _sys.modules[__name__]
 
+_basepath = _os.path.dirname(__file__)
+_filepath = _os.path.abspath(_os.path.join(_basepath, 'package.json'))
+with open(_filepath) as f:
+    package = json.load(f)
+
+js_package_name = package['name']
+py_package_name = __name__
+js_version = package['version']
 
 _js_dist = [
     {
         'relative_package_path': 'bundle.js',
         'external_url': (
-            'https://unpkg.com/dash-dangerously-set-inner-html@{}'
-            '/dash_dangerously_set_inner_html/bundle.js'
-        ).format(__version__),
-        'namespace': 'dash_dangerously_set_inner_html'
+            'https://unpkg.com/{}@{}/{}/bundle.js'
+        ).format(js_package_name, js_version, py_package_name),
+        'namespace': py_package_name
```

Comment:
This is similar to the logic in the dash-table -- it handles rcX and alphaX revisions much better (npm and pypi ver syntax is different an problematic with unpkg usage..)